packages/kit/src/runtime/client/client.js

```diff
@@ -11,7 +11,7 @@ import {
 	notifiable_store,
 	scroll_state
 } from './utils.js';
-import * as fetcher from './fetcher.js';
+import { increment, decrement, initial_fetch, native_fetch } from './fetcher.js';
```

Comment:
Maybe renaming `increment` and `decrement` to something else would make it clearer, at least locally if not in `fetcher.js`?

```suggestion
import { increment as lock_fetch, decrement as unlock_fetch, initial_fetch, native_fetch } from './fetcher.js';
```